ateam_ai/src/trajectory_generation/trapezoidal_motion_profile.cpp

```diff
@@ -193,9 +218,11 @@ Trajectory1d sample_trapezoidal_profile(
       // Phase 3
       double t_into_phase_3 = t - coeffs.phase_3.start_time;
       double d_into_phase_3 = coeffs.phase_3.start_dist;
+      double accel_direction = sign(coeffs.phase_3.end_vel - coeffs.phase_3.start_vel);
```

Comment:
Consider [`std::copysign`](https://en.cppreference.com/w/cpp/numeric/math/copysign) instead of custom lambda

```suggestion
      double accel_direction = std::copysign(1.0, coeffs.phase_3.end_vel - coeffs.phase_3.start_vel);
```